Package.swift

```diff
@@ -0,0 +1,86 @@
+// swift-tools-version:5.4.0
```

Comment:
I wanted to point out that we could _potentially_ lower this version to Swift 5.3 (or even lower), but I don't see any clear benefits. So we can leave this here. Does anyone have desire to reduce it?

Reply:
now it's 5.3 - it cant be lower according to using `.target()` method (in lower version some arguments are unavailable)

Reply:
ebb1f08aedafefd03c2800e3e86d4f9b665f5f8b

Reply:
Look's like you accidentally reverted it back to 5.4. 5.3 is needed to run in (our) github runners at the moment.

Reply:
here: 5fa3058ed5d53cbbc52e70d0125d74ce35c5170a